stdlib/influxdata/influxdb/sample/sample.flux

```diff
@@ -82,3 +98,25 @@ list = () => array.from(
         _setInfo(set: "usgs"),
     ],
 )
+
+// alignToNow shifts time values in input data to align the chronological last point to _now_.
+// When writing static historical sample datasets to **InfluxDB Cloud**, use alignToNow
+// to avoid losing sample data with timestamps outside of the retention period
+// associated with your InfluxDB Cloud account.
+//
+// ## Align sample data to now
+//
+// ```
+// import "influxdata/influxdb/sample"
+//
+// sample.data(set: "birdMigration")
+//    |> sample.alignToNow()
+// ```
+
+alignToNow = (tables=<-) => {
+    _lastTime = (tables |> keep(columns: ["_time"]) |> last(column: "_time") |> findRecord(fn: (key) => true, idx: 0))._time
```

Comment:
could we store the _lastTime for each dataset so we do not have to compute it again each time? Or does the data change enough that would get out of date often?

Reply:
That would save some compute time, but I don't know the best way to inform the function what dataset is actually being used. I guess we could introspect the measurement from the input data and use that to identify the specific set being used. As is, this is just a generic approach that can be used with any input data, not just sample data.

Reply:
I spec'd this approach out and have some reservations. This is what the measurement introspection approach I took:

```js
alignToNow = (tables=<-) => {
    _measurement = (tables |> findRecord(fn: (key) => true, idx:0))._measurement
    _endDates = [
        "machine": 2021-08-01T23:59:59.99999999Z,
        "airSensor": 2020-08-01T23:59:59.99999999Z
    ]
    _offset = int(v: now()) - int(v: dict.get(dict: _endDates, key: _measurement, default: now()))
    _offsetDuration = duration(v: _offset)

    return tables |> timeShift(duration: _offsetDuration)
}
```

This did prove to be _slightly_ more performant in my testing (5ms faster on average with an 11MB dataset), but there are a few things I don't love about it:

- We have to update the function anytime we add new sample datasets.
- We'd have to update the function with new dataset endDates if we decide the update the datasets (which does happen).
- The function will only work with the predefined datasets, so we lose some flexibility.

Thoughts?